monitoring/storage_linux_test.go

```diff
@@ -103,18 +103,20 @@ func (_ *StorageSuite) TestStorage(c *C) {
 
 	storageChecker{
 		StorageConfig: StorageConfig{
-			Path:          path.Join("/tmp", fmt.Sprintf("%d", time.Now().Unix())),
-			WillBeCreated: true,
-			HighWatermark: 40,
+			Path:              path.Join("/tmp", fmt.Sprintf("%d", time.Now().Unix())),
+			WillBeCreated:     true,
+			WatermarkCritical: 40,
```

Comment:
Not sure these values are useful. What about exercising happy path with warning < critical and disk usage reaching both warning, then criticial. Also warning >= critical which should yield an error, imo.